README.md

```diff
@@ -27,8 +27,12 @@ tart run ventura-vanilla
 3. Allow SSH. General -> Sharing -> Remote Login & Screen Sharing
 4. Display -> Advanced -> Prevent from sleeping
 5. Open Safari. Preferences -> Advanced -> Show Developer menu. Develop -> Allow Remote Automation.
-6. Sisable SIP. Run `tart run --recovery ventura-vanilla` -> Options -> Utilities menu -> Terminal -> `csrutil disable`.
```

Comment:
LGTM the sudoers part but for the SIP thing we have other reasons like https://github.com/cirruslabs/cirrus-ci-docs/issues/773. In general our main use case is for CI and CI jobs and tools love to have full access. Do you mind reverting the SIP part and maybe putting it as an optional last step?

Reply:
Bah, pesky CI wanting to _test_ things... 😆 

Sure, I'll put it back.

I wish there were a canonical list of things it prevents; I only ever learn one reason each place I work before then remembering to kill it.

Reply:
(restored, fixed typo)